.releaserc

```diff
@@ -1,5 +1,5 @@
 {
-  "branch": "master",
+  "branches": "master",
```

Comment:
Thanks for tagging me on this.  I need to update the PS node-sdk with this change.   Apparently "master" must be the default branch that it operates on (thankfully) :)

Reply:
Yes, "master" is the default so I actually thought about simply removing this line from the file. I only left it in in case we change the branch system down the line for some reason, I didn't want anyone to be confused when their branch wasn't found 🙂